src/App.jsx

```diff
@@ -14,15 +14,15 @@ const Button = styled.button`
   ${(props) =>
     props.primary &&
     css`
+      color: ${props.theme.colors.primary};
       background: palevioletred;
-      color: white;
     `}
 `;
 const Container = styled.div`
   text-align: center;
 `;
 
-function App() {
+export default function App() {
```

Comment:
Why not using arrow function here?

Reply:
why not? We can define the use or function or arrow functions for components. I do not have any special preference between the use of `function` or `const`

Reply:
Let's use arrow functions for the project